python/desc/sims/GCRCatSimInterface/InstanceCatalogWriter.py

```diff
@@ -385,19 +427,18 @@ def make_instcat_header(star_db, obs_md, outfile, object_catalogs=(),
     cat.phoSimHeaderMap = copy.deepcopy(DefaultPhoSimHeaderMap)
     cat.phoSimHeaderMap['nsnap'] = nsnap
     cat.phoSimHeaderMap['vistime'] = vistime
-    if imsim_catalog:
-        cat.phoSimHeaderMap['rawSeeing'] = ('rawSeeing', None)
-        cat.phoSimHeaderMap['FWHMgeom'] = ('FWHMgeom', None)
-        cat.phoSimHeaderMap['FWHMeff'] = ('FWHMeff', None)
-    else:
-        cat.phoSimHeaderMap['camconfig'] = 1
+    cat.phoSimHeaderMap['camconfig'] = 1
+
+    # the following commands are needed by imSim
+    cat.phoSimHeaderMap['rawSeeing'] = ('rawSeeing', None)
```

Comment:
These are not needed by imSim any longer since it will calculate these quantities if they aren't in the instance catalog.

Reply:
done